cogs/cog_inspire.py

```diff
@@ -24,34 +39,73 @@ async def display_quote(self, ctx):
 
         embed.add_field(name="Programmer", value=quote_split[0], inline=False)
         embed.add_field(name="Quote", value=quote_split[1], inline=False)
-        #currently sends to the author, however I want to send to user roles
-        await ctx.send(embed = embed)
 
+        # Get guild members, and send to members or author depending on true/false of check_param
+        server = ctx.guild
+        if check_param:
+            for role_id in role.split(' '):
+                for member in server.members:
+                    if role_id in [role.mention for role in member.roles]:
+                        await member.send(embed=embed)
+        else:
+            await ctx.send(embed=embed)
+
+    # This command gets inspirational quotes regardless of industry
     @commands.command(name="get-some-inspiration", aliases=["gsm"])
```

Comment:
Since the code between different kinds of inspiration commands is shared a lot, maybe we could make some helper functions to design the embeds, or even make a Builder pattern implementation.